packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -16,6 +16,17 @@ const pgDockerCompose = () => {
   return readFileSync(`${dockerFilesPath}/pg-docker-compose.yml`, 'utf8')
 }
 
+const generationConfig = `{
+  "create": true,
+  "update": true,
```

Comment:
We should keep `delete` as `true` and maybe `update` as `false`

Reply:
hmm.. Delete is very rarely used. Almost never - usually people have something like archive data. Data is very rarely deleted in production systems. 

Update is the main method being used.